src/components/FilterForTodos/FilterForTodos.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  filter: FilterStatus,
+  onFilterChange: (filter: FilterStatus) => void,
+};
+
+export const FilterForTodos: React.FC<Props> = ({
```

Comment:
Consider changing the component name to `FilterTodos` instead of `FilterForTodos` to follow the naming convention of the base filename (FilterForTodos.tsx). 🚀